benchmarks/backend/heatmap/insert_data-small_map.sh

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env bash
+
```

Comment:
Please describe what gets added. You can use this also in the test protocol.

Reply:
It is written in the script as comments. I don't want to use this for anything but the performance tests.
To use this for the test protocol would in my opinion be a misuse of the script. The manual test execution is already done, there is nothing that should now be changed about it (it points to a specific commit and works on that commit).

I will as a next step do the case study to verify if everything still works correctly and afterwards I will finish the benchmarks.

Reply:
I don't see a problem in reusing the same scripts. Finally, we want also the manual execution to be automated.

> it points to a specific commit and works on that commit

That is fine as long as it also works for later commits.

> I will as a next step do the case study to verify if everything still works correctly and afterwards I will finish the benchmarks.

:+1: